tests/list_repositories.yml

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: MIT
+---
+- name: subscription-manager repos --{{ mode | d("list") }}
+  shell: |
```

```suggestion
  shell: |
    set -euo pipefail
```
should be flagged by ansible-lint